core/src/main/java/kafka/server/builders/KafkaApisBuilder.java

```diff
@@ -0,0 +1,197 @@
+    }
+
+    public KafkaApisBuilder setMetadataCache(MetadataCache metadataCache) {
+        this.metadataCache = metadataCache;
+        return this;
+    }
+
+    public KafkaApisBuilder setMetrics(Metrics metrics) {
+        this.metrics = metrics;
+        return this;
+    }
+
+    public KafkaApisBuilder setAuthorizer(Optional<Authorizer> authorizer) {
+        this.authorizer = authorizer;
+        return this;
+    }
+
+    public KafkaApisBuilder setQuotas(QuotaManagers quotas) {
+        this.quotas = quotas;
+        return this;
+    }
+
+    public KafkaApisBuilder setFetchManager(FetchManager fetchManager) {
+        this.fetchManager = fetchManager;
+        return this;
+    }
+
+    public KafkaApisBuilder setBrokerTopicStats(BrokerTopicStats brokerTopicStats) {
+        this.brokerTopicStats = brokerTopicStats;
+        return this;
+    }
+
+    public KafkaApisBuilder setClusterId(String clusterId) {
+        this.clusterId = clusterId;
+        return this;
+    }
+
+    public KafkaApisBuilder setTime(Time time) {
+        this.time = time;
+        return this;
+    }
+
+    public KafkaApisBuilder setTokenManager(DelegationTokenManager tokenManager) {
+        this.tokenManager = tokenManager;
+        return this;
+    }
+
+    public KafkaApisBuilder setApiVersionManager(ApiVersionManager apiVersionManager) {
+        this.apiVersionManager = apiVersionManager;
+        return this;
+    }
+
+    public KafkaApis build() {
+        if (requestChannel == null) throw new RuntimeException("you must set requestChannel");
+        if (metadataSupport == null) throw new RuntimeException("you must set metadataSupport");
+        if (replicaManager == null) throw new RuntimeException("You must set replicaManager");
+        if (groupCoordinator == null) throw new RuntimeException("You must set groupCoordinator");
+        if (txnCoordinator == null) throw new RuntimeException("You must set txnCoordinator");
+        if (autoTopicCreationManager == null)
+            throw new RuntimeException("You must set autoTopicCreationManager");
+        if (config == null) config = new KafkaConfig(Collections.emptyMap());
+        if (configRepository == null) throw new RuntimeException("You must set configRepository");
+        if (metadataCache == null) throw new RuntimeException("You must set metadataCache");
+        if (metrics == null) throw new RuntimeException("You must set metrics");
+        if (quotas == null) throw new RuntimeException("You must set quotas");
+        if (fetchManager == null) throw new RuntimeException("You must set fetchManager");
+        if (brokerTopicStats == null) brokerTopicStats = new BrokerTopicStats();
+        if (apiVersionManager == null) throw new RuntimeException("You must set apiVersionManager");
+
+        return new KafkaApis(requestChannel,
+                             metadataSupport,
+                             replicaManager,
+                             groupCoordinator,
```

Comment:
Does this indentation follow our style guide? I can never remember what all we allow for continuations...

Reply:
Hmm, I'm not aware of any guidelines about continuations in the style guide.